Source/WTF/wtf/ObjectIdentifier.h

```diff
@@ -225,6 +225,28 @@ template<typename T, typename U, typename V, SupportsObjectIdentifierNullState s
     static bool isEmptyValue(const ObjectIdentifierGeneric<T, U, V, supportsNullState>& value) { return value.isHashTableEmptyValue(); }
 };
 
+template<typename T, typename U, typename V> struct HashTraits<ObjectIdentifierGeneric<T, U, V, SupportsObjectIdentifierNullState::No>> : SimpleClassHashTraits<ObjectIdentifierGeneric<T, U, V, SupportsObjectIdentifierNullState::No>> {
```

Comment:
if we did something like
```
using ValueType = ObjectIdentifierGeneric<T, U, V, SupportsObjectIdentifierNullState::No>;
```
Then we could make this easier to read. I don't see any down side.

Reply:
Ok.